src/components/Map/Map.js

```diff
@@ -45,8 +45,10 @@ export class Map extends Component {
       />
     ) : (
       <DynamicMap
-        containerElement={<div className={classes} onClick={this.onMapClicked} />}
```

Comment:
Where has the `onClick` gone?

Reply:
It wasn't used at all, there wasn't even a `onMapClicked` method that it was calling. It was probably copy pasted from `SearchMap` that had such method.